e2e/example_client/BUILD.bazel

```diff
@@ -13,3 +14,12 @@ go_library(
         "@io_bazel_rules_go//go/tools/bazel_testing:go_default_library",
     ],
 )
+
+go_bazel_test(
```

Comment:
The hope was that people would reuse `example_client` as a library and the other tests would live in `e2e/your_test_name` Could I ask you to `git mv` these up a dir? shouldn't be a painful move I hope